test-network-function/accesscontrol/suite.go

```diff
@@ -73,7 +73,8 @@ var _ = ginkgo.Describe(common.AccessControlTestKey, func() {
 
 //nolint:gocritic,funlen // ignore hugeParam error. Pointers to loop iterator vars are bad and `testCmd` is likely to be such.
 func runTestOnPods(env *config.TestEnvironment, testCmd testcases.BaseTestCase, testType string) {
-	ginkgo.It(fmt.Sprintf("Running Pod test : %s", testCmd.Name), func() {
+	testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestHostResourceIdentifier) + "-" + testType
```

Comment:
Did we loose  changes here? I previously commented on this, we should use testCmd.Name instead of testType

Reply:
not sure what happened. I fixed it.
The reason I want to also delay that ID fix is because I'm not sure how to handle this tests here.

Reply:
do we have those tests in the catalog.md btw ?

Reply:
yes,  but they are all clumped under the host-resource identifier.